src/seedsigner/views/seed_views.py

```diff
@@ -217,12 +217,24 @@ def __init__(self, cur_word_index: int = 0, is_calc_final_word: bool=False):
 
 
     def run(self):
+        is_final_word = (
+            self.cur_word_index == self.controller.storage.pending_mnemonic_length - 1 and
+            not self.is_calc_final_word and
```

Comment:
Naming convention on this bool gets confusing here.

`is_final_word` is True when it's not `is_calc_final_word`...

Also minor style nit: we almost always opt for testing conditions first, then setting the var as needed. It's just the tiniest bit harder to read when it's a var being set by the result of evaluating multiple conditions.